marathon_acme/clients/vault.py

```diff
@@ -39,6 +41,30 @@ def __init__(self, url, token, *args, **kwargs):
         super(VaultClient, self).__init__(*args, url=url, **kwargs)
         self._token = token
 
+    @classmethod
+    def from_env(cls, reactor=None, env=os.environ):
+        """
+        Create a Vault client with configuration from the environment. Supports
+        a limited number of the available config options:
+        https://github.com/hashicorp/vault/blob/v0.11.2/api/client.go#L28-L40
+        """
+        address = env.get('VAULT_ADDR', 'https://127.0.0.1:8200')
+        # This seems to be what the Vault CLI defaults to
+        token = env.get('VAULT_TOKEN', 'TEST')
+
+        # TODO: insecure = _get_environ_bool(env, 'VAULT_SKIP_VERIFY')
```

Comment:
This comment is a bit confusing. Is it a TODO for implementing an insecure mode where we don't verify certs?